### ISSUE TITLE
deCONZ 6.15 very unstable in HA 2022.8 - watchdog does not restart it.

### ISSUE THREAD

**Issue by cortado** [home-assistant/addons#2621]
### Describe the issue you are experiencing

 Stops at least once every day, and Watchdog does not restart it (some other plugins also had that issue in HA 2022.8, but was patched. so missing patch?)

deConz have been rock solid since i started with Conbee II and HA. Not anymore! Witch is very troublesome for all security sensors and lights. 

Any solution? Logs does not tell me much, and I am not a developer. Therefor run everything as vanilla as possible. So no log entries from deCONZ that gives any hint, but a lot of errors from websocket, and Supervisor also gets timeouts – so there seems to be many problems with this version of HA

Downgrade to 2022.7?

 

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

deCONZ

### What is the version of the add-on?

6.15.0

### Steps to reproduce the issue

1.
2.
3.
...

### Additional information

**System:**
RPib+ (32)
Home Assistant 2022.8.6
Supervisor 2022.08.3
Operating System 8.4
Grensesnitt 20220802.0 – latest

**Conbee II**
2.17.01 / 1.7.2022
Firmware
26720700

**Comment by davidrustingha** [comment]
I'm having the same problems. After a host reboot it works for a couple of hours, and then first only inside deconz it works, and later deconz becomes unresponsive too.

**Comment by aerox845** [comment]
for me its not working at all anymore somehow it fully stopped working after updating - to HA 2022.8  -  
 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2257841-if00 / serialno: DE2XXXXX, ConBee II
13:10:59:260 start reconnect to network
13:11:02:128 device state timeout (handled)

and pretty much tried alls the fixes - even updated the stick FW

**Comment by mdegat01** [comment]
Can you share your [system health](https://my.home-assistant.io/redirect/system_health/) report? Just copy and paste it from the link.

Also can you go to your list of [integrations](https://my.home-assistant.io/redirect/integrations/) and do "download diagnostics" from the deconz integration and share that?

It would also help if you shared the logs from the deconz addon. And the ones from supervisor you saw about timeouts.

**Comment by aerox845** [comment]
system health:

## System Information

version | core-2022.8.6
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.10.5
os_name | Linux
os_version | 5.15.55
arch | x86_64
timezone | Europe/Berlin
config_dir | /config

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 8.4
-- | --
update_channel | stable
supervisor_version | supervisor-2022.08.3
agent_version | 1.2.1
docker_version | 20.10.14
disk_total | 87.9 GB
disk_used | 17.4 GB
healthy | true
supported | true
board | ova
supervisor_api | ok
version_api | ok
installed_addons | ESPHome (2021.12.3), TasmoAdmin (0.20.0), Node-RED (13.3.2), File editor (5.3.3), Grafana (7.6.0), Samba share (10.0.0), Studio Code Server (5.3.0), deCONZ (6.15.0)

</details>

<details><summary>Dashboards</summary>

dashboards | 1
-- | --
resources | 0
views | 3
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 13. August 2022 um 06:48
-- | --
current_recorder_run | 22. August 2022 um 14:06
estimated_db_size | 234.41 MiB
database_engine | sqlite
database_version | 3.38.5

</details>



[config_entry-deconz-78954d013b0d11ebbe94d30b645c482e.json.txt](https://github.com/home-assistant/addons/files/9400605/config_entry-deconz-78954d013b0d11ebbe94d30b645c482e.json.txt)

10:10:08:963 Skip idle timer callback, too early: elapsed 856 msec
10:10:10:104 Announced to internet https://phoscon.de/discover
10:10:11:383 device state timeout (handled)
10:10:11:464 void zmMaster::handleStateIdle(zmMaster::MasterEvent) not connected goto OFF state
10:10:12:963 DEV Tick.Init: booted after 8000 seconds
[10:10:15] INFO: Successfully send discovery information to Home Assistant.
10:10:17:975 New websocket 172.30.32.1:40438 (state: 3) 
10:10:19:972 GW update firmware found: /usr/share/deCONZ/firmware/deCONZ_ConBeeII_0x26720700.bin.GCF
10:10:25:471 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2257841-if00 / serialno: DE2257841, ConBee II
10:10:26:963 start reconnect to network
10:10:28:807 device state timeout (handled)
10:10:28:887 void zmMaster::handleStateIdle(zmMaster::MasterEvent) not connected goto OFF state
10:10:31:963 failed to reconnect to network try=1
10:10:36:963 failed to reconnect to network try=2
10:10:41:963 failed to reconnect to network try=3
10:10:43:471 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2257841-if00 / serialno: DE2257841, ConBee II
10:10:44:472 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2257841-if00 / serialno: DE2257841, ConBee II
10:10:46:963 failed to reconnect to network try=4
10:10:48:471 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2257841-if00 / serialno: DE2257841, ConBee II


i cant find the logs from deconz addon hmm

**Comment by cortado** [comment]
## System Information

version | core-2022.8.6
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.10.5
os_name | Linux
os_version | 5.15.32-v7
arch | armv7l
timezone | Europe/Oslo
config_dir | /config

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 8.4
-- | --
update_channel | stable
supervisor_version | supervisor-2022.08.3
agent_version | 1.2.1
docker_version | 20.10.14
disk_total | 57.8 GB
disk_used | 11.1 GB
healthy | true
supported | true
board | rpi3
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.3.3), Node-RED (13.3.2), Mosquitto broker (6.1.2), Samba share (10.0.0), Duck DNS (1.15.0), SSH & Web Terminal (12.0.2), deCONZ (6.15.0), chrony (2.4.0), phpMyAdmin (0.8.1), Log Viewer (0.14.0), MariaDB (2.5.1), Z-Wave JS (0.1.65), Filebrowser (2.22.4), Z-Wave JS to MQTT (0.45.0), Cloudflared (2.0.13), Zigbee2MQTT (1.27.0-1)

</details>

<details><summary>Dashboards</summary>

dashboards | 1
-- | --
resources | 1
views | 10
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 19. august 2022, 19:06
-- | --
current_recorder_run | 22. august 2022, 15:59
estimated_db_size | 483.28 MiB
database_engine | mysql
database_version | 10.6.8

</details>

## deCONZ diagnostics (not all of the file it was too long..)
 "integration_manifest": {
    "domain": "deconz",
    "name": "deCONZ",
    "config_flow": true,
    "documentation": "https://www.home-assistant.io/integrations/deconz",
    "requirements": [
      "pydeconz==102"
    ],
    "ssdp": [
      {
        "manufacturer": "Royal Philips Electronics",
        "manufacturerURL": "http://www.dresden-elektronik.de"
      }
    ],
    "codeowners": [
      "@Kane610"
    ],
    "quality_scale": "platinum",
    "iot_class": "local_push",
    "loggers": [
      "pydeconz"
    ],
    "is_built_in": true
  },
  "data": {
    "config": {
      "entry_id": "aacfed5820894a7d8c48a44b71b9dfc3",
      "version": 1,
      "domain": "deconz",
      "title": "00212E048ACA",
      "data": {
        "api_key": "**REDACTED**",
        "host": "core-deconz",
        "port": 40850
      },
      "options": {
        "allow_clip_sensor": true,
        "allow_deconz_groups": false,
        "master": true,
        "allow_new_devices": true
      },
      "pref_disable_new_entities": false,
      "pref_disable_polling": false,
      "source": "hassio",
      "unique_id": "**REDACTED**",
      "disabled_by": null
    },
    "deconz_config": {
      "UTC": "2022-08-22T14:00:46",
      "apiversion": "1.16.0",
      "backup": {
        "errorcode": 0,
        "status": "idle"
      },
      "bridgeid": "**REDACTED**",
      "datastoreversion": "93",
      "devicename": "ConBee II",
      "dhcp": true,
      "disablePermitJoinAutoOff": false,
      "factorynew": false,
      "fwversion": "0x26720700",
      "gateway": "172.30.33.1",
      "internetservices": {
        "internet": "connected",
        "remoteaccess": "disconnected",
        "swupdate": "connected",
        "time": "connected"
      },
      "ipaddress": "172.30.33.4",
      "lightlastseeninterval": 60,
      "linkbutton": false,
      "localtime": "2022-08-22T16:00:46",
      "mac": "**REDACTED**",
      "modelid": "deCONZ",
      "name": "Phoscon-GW",
      "netmask": "255.255.254.0",
      "networkopenduration": 180,
      "panid": "**REDACTED**",
      "portalconnection": "disconnected",
      "portalservices": false,
      "portalstate": {
        "communication": "disconnected",
        "incoming": false,
        "outgoing": false,
        "signedon": false
      },
      "proxyaddress": "none",
      "proxyport": 0,
      "replacesbridgeid": null,
      "rfconnected": true,
      "starterkitid": "",
      "swupdate": {
        "checkforupdate": false,
        "devicetypes": {
          "bridge": false,
          "lights": [],
          "sensors": []
        },
        "notify": false,
        "text": "",
        "updatestate": 0,
        "url": ""
      },
      "swupdate2": {
        "autoinstall": {
          "on": false,
          "updatetime": ""
        },
        "bridge": {
          "lastinstall": "2022-07-01T13:20:49",
          "state": "noupdates"
        },
        "checkforupdate": false,
        "lastchange": "",
        "state": "noupdates"
      },
      "swversion": "2.17.1",
      "timeformat": "24h",
      "timezone": "Etc/UTC",
      "uuid": "1ba851d5-96a4-4b46-88e2-82dc508a94b0",
      "websocketnotifyall": true,
      "websocketport": 8081,
      "whitelist": {
        "00B97F5CF7": {
          "create date": "2021-01-24T14:00:59",
          "last use date": "2021-01-24T14:05:27",
          "name": "Phoscon#B375x728"
        },

**Comment by simonfox770** [comment]
Also experiencing this. The DeCONZ add-on loads and then enters into a loop of reconnecting to the network.

```
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/firmware.sh
[17:44:04] INFO: GCFFlasher V3_17 (c) dresden elektronik ingenieurtechnik gmbh
Path             | Vendor | Product | Serial     | Type
-----------------+--------+---------+------------+-------
/dev/ttyACM0     | 0x1CF1 | 0x0030  |            | ConBee II 
/dev/ttyACM1     | 0x1CF1 | 0x0030  | DE2420351  | ConBee II 
cont-init: info: /etc/cont-init.d/firmware.sh exited 0
cont-init: info: running /etc/cont-init.d/nginx.sh
cont-init: info: /etc/cont-init.d/nginx.sh exited 0
cont-init: info: running /etc/cont-init.d/novnc.sh
cont-init: info: /etc/cont-init.d/novnc.sh exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun deconz (no readiness notification)
services-up: info: copying legacy longrun nginx (no readiness notification)
services-up: info: copying legacy longrun otau-deconz (no readiness notification)
services-up: info: copying legacy longrun otau-ikea (no readiness notification)
services-up: info: copying legacy longrun otau-ledvance (no readiness notification)
services-up: info: copying legacy longrun websockify (no readiness notification)
s6-rc: info: service legacy-services successfully started
[17:44:04] INFO: Websockify waiting for VNC to start
[17:44:04] INFO: Running the IKEA OTA updater...
[17:44:04] INFO: Running the deCONZ OTA updater...
[17:44:04] INFO: Running the OSRAM LEdvance OTA updater...
[17:44:05] INFO: Starting VNC server (local/yes)...
[17:44:05] INFO: Starting websockify...
WebSocket server settings:
  - Listen on 127.0.0.1:5901
  - Flash security policy server
- Web server. Web root: /usr/share/novnc
  - No SSL/TLS support (no cert file)
  - proxying from 127.0.0.1:5901 to 127.0.0.1:5900
[17:44:08] INFO: deCONZ waiting for VNC to start
[17:44:08] INFO: Starting the deCONZ gateway...
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
[17:44:12] INFO: Starting Nginx...
2022/08/27 17:44:12 [notice] 119#119: using the "epoll" event method
2022/08/27 17:44:12 [notice] 119#119: nginx/1.14.2
2022/08/27 17:44:12 [notice] 119#119: OS: Linux 5.15.60
2022/08/27 17:44:12 [notice] 119#119: getrlimit(RLIMIT_NOFILE): 1048576:1048576
2022/08/27 17:44:12 [notice] 119#119: start worker processes
2022/08/27 17:44:12 [notice] 119#119: start worker process 1167
17:44:12:124 HTTP Server listen on address 0.0.0.0, port: 40850, root: /usr/share/deCONZ/webapp/
17:44:12:527 /data/.local/share/dresden-elektronik/deCONZ/config.ini exists and is writeable
17:44:12:527 /data/.local/share/dresden-elektronik/deCONZ/zll.db exists and is writeable
17:44:12:634 COM: use stable device path /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2420351-if00
17:44:12:726 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2420351-if00 / serialno: , ConBee II
17:44:12:728 ZCLDB init file /data/.local/share/dresden-elektronik/deCONZ/zcldb.txt
17:44:12:881 DDF enabled for Gold status
17:44:13:115 parent process s6-supervise
17:44:13:115 gw run mode: docker/hassio
17:44:13:115 GW sd-card image version file does not exist: /data/.local/share/dresden-elektronik/deCONZ/gw-version
17:44:13:119 DB sqlite version 3.27.2
17:44:13:120 DB PRAGMA page_count: 57
17:44:13:120 DB PRAGMA page_size: 4096
17:44:13:120 DB PRAGMA freelist_count: 3
17:44:13:120 DB file size 233472 bytes, free pages 3
17:44:13:120 DB PRAGMA user_version: 9
17:44:13:120 DB cleanup
17:44:13:121 DB create temporary views
17:44:13:134 DB legacy loading sensor Solarium Button 25, later handled by DDF Samsung Button (IM6001-BTP01)
17:44:13:145 DB legacy loading sensor Temperature 26 26, later handled by DDF Samsung Button (IM6001-BTP01)
17:44:13:157 Started websocket server on 0.0.0.0, port: 8081
17:44:13:162 [INFO] - Found file containing button maps. Parsing data...
17:44:13:171 [INFO] - Button maps loaded.
17:44:13:173 found node plugin: libde_rest_plugin.so - REST API Plugin
17:44:13:174 found node plugin: libde_signal_plugin.so - Signal Monitor Plugin
17:44:20:191 found node plugin: libstd_otau_plugin.so - STD OTAU Plugin
17:44:20:281 LightNode 1: Configuration tool 1 added

---devices load here---

17:44:20:943 dlg action: Read binding table
17:44:21:006 DEV found DDF for 0x286D9700010C083E, path: 
17:44:21:075 New websocket 172.30.32.1:60290 (state: 3) 
17:44:21:079 DEV no DDF for 0x90FD9FFFFE32BFC6, modelId: TRADFRI control outlet
17:44:21:079 DEV create on-the-fly DDF for 0x90FD9FFFFE32BFC6
17:44:21:080 DEV no DDF for 0x841826000006495F, modelId: LIGHTIFY BR RGBW
17:44:21:080 DEV create on-the-fly DDF for 0x841826000006495F
17:44:21:080 DEV found DDF for 0x8418260000086930, path: 
17:44:21:080 DEV no DDF for 0xB0CE18140309E44B, modelId: E12-N14
17:44:21:080 DEV create on-the-fly DDF for 0xB0CE18140309E44B
17:44:21:081 DEV found DDF for 0xB0CE1814030A0CE5, path: 
17:44:21:081 DEV no DDF for 0xF0D1B80000019542, modelId: BR30 RGBW
17:44:21:081 DEV create on-the-fly DDF for 0xF0D1B80000019542
17:44:21:081 DEV found DDF for 0x8418260000065823, path: 
17:44:21:081 DEV no DDF for 0x000D6F000AAED7DF, modelId: 4257050-RZHAC
17:44:21:081 DEV create on-the-fly DDF for 0x000D6F000AAED7DF
17:44:21:082 DEV no DDF for 0xF0D1B800000526D7, modelId: PLUG
17:44:21:082 DEV create on-the-fly DDF for 0xF0D1B800000526D7
17:44:21:082 DEV found DDF for 0x841826000008AFCF, path: 
17:44:21:309 DEV Tick.Init: booted after 8000 seconds
17:44:21:776 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2420351-if00 / serialno: , ConBee II
17:44:22:020 Announced to internet https://phoscon.de/discover
[17:44:22] INFO: Successfully send discovery information to Home Assistant.
17:44:22:810 Skip idle timer callback, too early: elapsed 877 msec
17:44:25:193 device state timeout (handled)
17:44:25:274 void zmMaster::handleStateIdle(zmMaster::MasterEvent) not connected goto OFF state
17:44:39:849 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2420351-if00 / serialno: , ConBee II
17:44:40:809 Skip idle timer callback, too early: elapsed 705 msec
17:44:41:351 GW update firmware found: /usr/share/deCONZ/firmware/deCONZ_ConBeeII_0x26720700.bin.GCF
17:44:41:810 start reconnect to network
17:44:43:521 device state timeout (handled)
17:44:43:601 void zmMaster::handleStateIdle(zmMaster::MasterEvent) not connected goto OFF state
17:44:47:061 failed to reconnect to network try=1
17:44:52:309 failed to reconnect to network try=2
17:44:57:309 failed to reconnect to network try=3
17:44:57:849 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2420351-if00 / serialno: , ConBee II
17:44:58:810 Skip idle timer callback, too early: elapsed 782 msec
17:45:01:297 device state timeout (handled)
17:45:01:377 void zmMaster::handleStateIdle(zmMaster::MasterEvent) not connected goto OFF state
17:45:02:309 failed to reconnect to network try=4
17:45:07:310 failed to reconnect to network try=5
17:45:12:309 failed to reconnect to network try=6
17:45:15:847 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2420351-if00 / serialno: , ConBee II
17:45:16:809 Skip idle timer callback, too early: elapsed 783 msec
17:45:17:309 failed to reconnect to network try=7
17:45:19:289 device state timeout (handled)
17:45:19:369 void zmMaster::handleStateIdle(zmMaster::MasterEvent) not connected goto OFF state
17:45:22:310 failed to reconnect to network try=8
17:45:27:310 failed to reconnect to network try=9
17:45:32:310 failed to reconnect to network try=10
17:45:33:848 COM: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2420351-if00 / serialno: , ConBee II
17:45:34:809 Skip idle timer callback, too early: elapsed 821 msec
17:45:37:257 device state timeout (handled)
17:45:37:310 reconnect network failed, try later
17:45:37:337 void zmMaster::handleStateIdle(zmMaster::MasterEvent) not connected goto OFF state
17:45:37:810 start reconnect to network
17:45:43:060 failed to reconnect to network try=1
17:45:48:309 failed to reconnect to network try=2
```

**Comment by simonfox770** [comment]
Tonight I did the following:
Rolled back DeCONZ add-on from 6.15 to 6.13 (Running Core 2022.8.6): Same problem.
Rolled back Core to 2022.6.7 (Still using rolled back DeCONZ 6.13): Same problem.

This setup was definitely working at the times the backups were created. The only thing I can't roll back is Home Assistant OS, making me think the problem lies in there somewhere.

**Comment by mdegat01** [comment]
@cortado it looks like you have both Deconz and Z2M installed. Are you trying to have them both use the same conbee ii stick? If so that won't work. You cannot share the same zigbee stick between two zigbee services

**Comment by mdegat01** [comment]
@aerox845 hrm nothing jumping out at me. By any chance do you also have ZHA added to HA and trying to use the same conbee ii stick as Deconz? If so that won't work for the same reasons I just said for cortado above.

**Comment by aerox845** [comment]
@mdegat01  ich got it fixed myself and i guess its the same issue for nearly everyone here:

i am using Unraid as Server and got Homeassistant running in a VM - i updated everything the last weekend which was a mistake in doing a big "upgrade all" and got confused what caused it to stop working.

but for everyone that gets into a "connect network, reconnect loop" the Linux Kernel the new Unraid OS uses is faulty with passing thru USB-Devices to the VM. even in the newes Unraid beta this isnt fixed! there are workarounds but i just downgraded back to Unraid 6.9.2 and everything instantly started working like a charm again.

`Linux kernel:
version 5.10.28 <- for Unraid 6.9.2` vs unraid 6.10.2:
`Linux kernel:
version 5.15.43-Unraid (CVE-2022-21499) <- VM Passthru stopped working`

**Comment by cortado** [comment]
I did however discover one thing: I had an automation set up in the Phoscon app, connecting a motion sensor to a light directly (other automations are in Node Red) – when I deleted that, it stopped crashing every night. Could there be a bug?  Also: rest of HA become almost as stable as before with 2022.8.7 and updated Supervisor. After these 3 changes I have not had deCONZ crash.

**Comment by simonfox770** [comment]
So I noticed my Home Assistant's Docker interface is at 172.30.32.1 while the Phoscon interface shows the deCONZ Docker interface is running on 172.30.3**3**.5. Could this be related? I've found a few mentions of weird Docker networking issues giving similar problems (https://community.home-assistant.io/t/after-update-home-assistant-error-connecting-to-deconz-gateway/326660). Is this as it should be?

(and yes the solution in that thread - making sure the config is looking at "core-deconz" - is what is on my install.

**Comment by simonfox770** [comment]
I can't get my ConBeeII to work on other systems (a RPi running the official Phoscon image nor the Windows app) and I'm starting to think it's just dead. Oh well.

**Comment by mdegat01** [comment]
> I had an automation set up in the Phoscon app, connecting a motion sensor to a light directly (other automations are in Node Red) – when I deleted that, it stopped crashing every night

It's definitely possible. To be honest I'm not very familiar with the actual Phoscon app, I only know addons and HA. If it is an issue with the Phoscon app itself you should submit a bug [here](https://github.com/dresden-elektronik/deconz-rest-plugin/issues).

> rest of HA become almost as stable as before with 2022.8.7 and updated Supervisor.

This is definitely possible. There were a few significant issues with addons resolved by 2022.08.5 of supervisor. Particularly immediately following a host reboot, addons were sometimes unable to get their config. So if the timing of this issue was matching up with a host reboot then there's a decent chance the supervisor update took care of it.

**Comment by simonfox770** [comment]
Managed to fix the problem by connecting the ConBeeII to my PC and using the command line tool to upgrade the firmware.
